### ISSUE TITLE
Menu text renders & as &amp;

### ISSUE THREAD

**Issue by roballred** [OrchardCMS/Orchard#7852]
I know this may seem trivial but I think this needs to be fixed for good out of box experience. Content Menu Item's menu text will render the text "Home & Garden" as "Home `&amp;` Garden" on the menu generated by this content. I think it is incorrect for it to work this way and should by default be rendered as it looks in the Edit view of the menu item. When trying to tell people who are non technical end users that it cannot by default render this correctly it looks bad for orchard and is a bug in my opinion.

**Comment by mdameer** [comment]
I can't reproduce this problem on latest version of Orchard, can you please provide more info about this bug, or include some shots

**Comment by roballred** [comment]
My markdown was displaying the example incorrectly. I fixed it in my post. I will provide a screen shot of the problem. I have attached a screen cap of this problem happening with accents on a spanish page we have. the url for this page is https://esd.wa.gov/espanol, but it might get changed to prevent this from happening so I am attaching a screen cap of the problem.
![l_ce77 tmp](https://user-images.githubusercontent.com/549504/35643986-b046a3d0-067c-11e8-98d7-9dd0ff27f458.PNG)
